src/packages/v4/components/modals/V4TokenHoldersModal/HoldersList.tsx

```diff
@@ -10,42 +10,44 @@ import EthereumAddress from 'components/EthereumAddress'
 import Loading from 'components/Loading'
 import { TokenAmount } from 'components/TokenAmount'
 import { JuiceListbox } from 'components/inputs/JuiceListbox'
-import { NativeTokenValue } from 'juice-sdk-react'
-import { OrderDirection, Participant_OrderBy, ParticipantsDocument } from 'packages/v4/graphql/client/graphql'
-import { useSubgraphQuery } from 'packages/v4/graphql/useSubgraphQuery'
+import { useParticipantsQuery } from 'generated/v4/graphql'
+import { NativeTokenValue, useJBChainId } from 'juice-sdk-react'
+import { bendystrawClient } from 'lib/apollo/bendystrawClient'
 import { useEffect, useState } from 'react'
 import { formatPercent } from 'utils/format/formatNumber'
 import { tokenSymbolText } from 'utils/tokenSymbolText'
 import { DownloadTokenHoldersModal } from './DownloadTokenHoldersModal'
 
-interface ParticipantOption {
-  label: string
-  value: Participant_OrderBy
+type Participant = {
+  volume: bigint
+  lastPaidTimestamp: number
+  balance: bigint
+  creditBalance: bigint
+  address: string
 }
 
-type Participant = {
-  volume: bigint;
-  lastPaidTimestamp: number;
-  balance: bigint;
-  stakedBalance: bigint;
-  id: string;
-  wallet: {
-      id: string;
-  };
+type OrderBy = keyof Pick<
+  Participant,
+  'volume' | 'lastPaidTimestamp' | 'balance'
+>
+
+interface ParticipantOption {
+  label: string
+  value: OrderBy
 }
 
 const participantOptions = (tokenText: string): ParticipantOption[] => [
   {
     label: t`${tokenText} balance`,
-    value: Participant_OrderBy.balance,
+    value: 'balance',
```

Comment:
i miss the unumerated OrderBys from the subgraph, but here we are. not type strict unfortunately